src/parse.rs

```diff
@@ -39,6 +39,7 @@ const FLAG_DOTNL: u32 = 1 << 2;
 const FLAG_SWAP_GREED: u32 = 1 << 3;
 const FLAG_IGNORE_SPACE: u32 = 1 << 4;
 const FLAG_UNICODE: u32 = 1 << 5;
+const FLAG_CRLF: u32 = 1 << 6;
```

Comment:
flags have been moved to a new file, `flags.rs` since this PR was made, please can you rebase/merge latest main branch? :)